### PR TITLE
Edit credentials markdown

### PR DESCRIPTION
**Pull request by willrayeo**
After feedback, the markdown cells associated with generation of credentials has been updated to make this clearer. 

In addition, the black code formatter was used when making changes locally. Therefore, the formatting in the notebooks has been standardised as per the repository instructions.

### CODE REVIEW

File: sentinelhub/lake_ice_monitoring.ipynb [hunk 1]
```diff
@@ -58,11 +58,11 @@
...
     "\n",
-    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you will need to create a profile specific to the Copernicus Data Space Ecosystem. You can obtain your `CLIENT ID` and `CLIENT SECRET` by generating them in your Dashboard before being prompted to enter them in the following cell:"
+    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you should create a profile specific to the Copernicus Data Space Ecosystem, you can do this in the following cell:"
```
Comment (Kampen1988) [inline sentinelhub/lake_ice_monitoring.ipynb:65]: I'd split the sentence after Ecosystem with a full stop.

File: sentinelhub/lake_ice_monitoring.ipynb [hunk 2]
```diff
@@ -58,11 +58,11 @@
...
     "\n",
-    "Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. \n",
+    "Now you have your `client_id` & `client_secret` it is recommended to configure a new profile in your Sentinel Hub Python package. Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. This is useful as the config class is usually only temporary in your notebook and by saving the configuration to your profile you won't need to generate new credentials each time you rerun or write a new Jupyter Notebook. \n",
```
[the remark this answers is not in the document]
Comment (Kampen1988) [reply sentinelhub/lake_ice_monitoring.ipynb:63]: "This is useful as **changes to** the the config class **are** usually only temporary in your notebook and by saving the configuration to your profile you won't need to generate new credentials **or overwrite/change the default profile** each time you rerun or write a new Jupyter Notebook"

File: sentinelhub/migration_from_scihub_guide.ipynb [hunk 3]
```diff
@@ -397,11 +411,11 @@
...
     "\n",
-    "Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. \n",
+    "Now you have your `client_id` & `client_secret` it is recommended to configure a new profile in your Sentinel Hub Python package. Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. This is useful as the config class is usually only temporary in your notebook and by saving the configuration to your profile you won't need to generate new credentials each time you rerun or write a new Jupyter Notebook. \n",
```
Comment (Kampen1988) [inline sentinelhub/migration_from_scihub_guide.ipynb:416]: See above

File: sentinelhub/migration_from_scihub_guide.ipynb [hunk 4]
```diff
@@ -397,11 +411,11 @@
...
     "\n",
-    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you will need to create a profile specific to the Copernicus Data Space Ecosystem. You can obtain your `CLIENT ID` and `CLIENT SECRET` by generating them in your Dashboard before being prompted to enter them in the following cell:"
+    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you should create a profile specific to the Copernicus Data Space Ecosystem, you can do this in the following cell:"
```
Comment (Kampen1988) [inline sentinelhub/migration_from_scihub_guide.ipynb:418]: See above

File: sentinelhub/pollution.ipynb [hunk 5]
```diff
@@ -76,11 +76,11 @@
...
     "\n",
-    "Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. \n",
+    "Now you have your `client_id` & `client_secret` it is recommended to configure a new profile in your Sentinel Hub Python package. Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. This is useful as the config class is usually only temporary in your notebook and by saving the configuration to your profile you won't need to generate new credentials each time you rerun or write a new Jupyter Notebook. \n",
```
Comment (Kampen1988) [inline sentinelhub/pollution.ipynb:81]: See above

File: sentinelhub/pollution.ipynb [hunk 6]
```diff
@@ -76,11 +76,11 @@
...
     "\n",
-    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you will need to create a profile specific to the Copernicus Data Space Ecosystem. You can obtain your `CLIENT ID` and `CLIENT SECRET` by generating them in your Dashboard before being prompted to enter them in the following cell:"
+    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you should create a profile specific to the Copernicus Data Space Ecosystem, you can do this in the following cell:"
```
Comment (Kampen1988) [inline sentinelhub/pollution.ipynb:83]: See above

File: sentinelhub/process_request.ipynb [hunk 7]
```diff
@@ -64,11 +64,11 @@
...
     "\n",
-    "Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. \n",
+    "Now you have your `client_id` & `client_secret` it is recommended to configure a new profile in your Sentinel Hub Python package. Instructions on how to configure your Sentinel Hub Python package can be found [here](https://sentinelhub-py.readthedocs.io/en/latest/configure.html). Using these instructions you can create a profile specific to using the package for accessing Copernicus Data Space Ecosystem data collections. This is useful as the config class is usually only temporary in your notebook and by saving the configuration to your profile you won't need to generate new credentials each time you rerun or write a new Jupyter Notebook. \n",
```
Comment (Kampen1988) [inline sentinelhub/process_request.ipynb:69]: See above

File: sentinelhub/process_request.ipynb [hunk 8]
```diff
@@ -64,11 +64,11 @@
...
     "\n",
-    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you will need to create a profile specific to the Copernicus Data Space Ecosystem. You can obtain your `CLIENT ID` and `CLIENT SECRET` by generating them in your Dashboard before being prompted to enter them in the following cell:"
+    "If you are a first time user of the Sentinel Hub Python package for Copernicus Data Space Ecosystem, you should create a profile specific to the Copernicus Data Space Ecosystem, you can do this in the following cell:"
```
Comment (Kampen1988) [inline sentinelhub/process_request.ipynb:71]: See above

**Review by Kampen1988 — CHANGES_REQUESTED** [review]

File: sentinelhub/lake_ice_monitoring.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (willrayeo) [reply sentinelhub/lake_ice_monitoring.ipynb:63]: Agreed, both changes made.

**Comment by willrayeo** [comment]
Requested changes made to all notebooks in the Sentinel Hub folder.

**Pull request merged**